javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/OpenTelemetryInstaller.java

```diff
@@ -25,13 +26,11 @@ public final class OpenTelemetryInstaller {
    *
    * @return the {@link AutoConfiguredOpenTelemetrySdk}
    */
-  @SuppressWarnings("deprecation") // Config usage, to be removed
-  static AutoConfiguredOpenTelemetrySdk installOpenTelemetrySdk(
-      io.opentelemetry.instrumentation.api.config.Config config) {
+  static AutoConfiguredOpenTelemetrySdk installOpenTelemetrySdk() {
     AutoConfiguredOpenTelemetrySdkBuilder builder =
         AutoConfiguredOpenTelemetrySdk.builder()
             .setResultAsGlobal(true)
-            .addPropertiesSupplier(config::getAllProperties);
+            .addPropertiesSupplier(new ConfigurationFileLoader());
```

Comment:
```suggestion
            .addPropertiesSupplier(ConfigurationFileLoader::new);
```
❓

Reply:
Wouldn't that result in a `Supplier<Supplier<Map<String, String>>>`?

Reply:
Yeah, you're right. Ignore me.